### PR TITLE
Add Indonesian translation

### PR DESCRIPTION
**Pull request by imanudinsh**
### Changes
Add Indonesian translation

### Testing
- [x] Tested on a physical device
- [ ] Added or modified unit test cases

### Others
- Fixes #0
- Connects #200

### CODE REVIEW

File: fastlane/metadata/android/in-ID/short_description.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Santai, tingkatkan fokus, dan tingkatkan produktivitas dengan derau pelataran minimal.
```
Comment (ashutoshgngwr) [inline fastlane/metadata/android/in-ID/short_description.txt:1]: The short description needs to be 80 characters at most. It's 86 right now. Can you please rephrase it to be <= 80?

File: app/src/main/res/values-in-rID/strings.xml [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  <string name="dismiss">Singkirkan</string>
+  <string name="save_preset">Simpan Preset</string>
+  <string name="name">Nama</string>
+  <string name="save">Simpan</string>
+  <string name="cancel">Batal</string>
+  <string name="preset_name_cannot_be_empty">Nama preset tidak boleh kosong!</string>
+  <string name="preset_already_exists">Preset dengan nama yang diberikan sudah ada!</string>
+  <string name="preset_saved">Preset disimpan!</string>
+  <string name="saved_presets">Preset Tersimpan</string>
+  <string name="preset_delete_confirmation">Konfirmasi hapus% s?</string>
```
Comment (ashutoshgngwr) [inline app/src/main/res/values-in-rID/strings.xml:66]: There's a typo here.

```suggestion
  <string name="preset_delete_confirmation">Konfirmasi hapus %s?</string>
```

File: app/src/main/res/values-in-rID/strings.xml [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    Anda belum membuat preset apa pun. Untuk membuat preset, buka Perpustakaan dan putar suara.
+    Kemudian klik tombol Simpan yang muncul di sudut bawah layar.
+</string>
+  <string name="sleep_timer">Pengatur Waktu Tidur</string>
+  <string name="sleep_timer_description">
+    Noice akan mati sendiri ketika timer berakhir. Gunakan tombol di bawah untuk menambahkan waktu dan memulai
+    pengatur waktu.
+</string>
+  <string name="reset">Setel ulang</string>
+  <string name="auto_sleep_schedule_cancelled">Pengatur waktu tidur otomatis dibatalkan!</string>
```
Comment (ashutoshgngwr) [inline app/src/main/res/values-in-rID/strings.xml:79]: Google translates it to _Sleep timer automatically canceled!_ but it needs to imply _Sleep timer cancelled!_ or _Automatic sleep timer cancelled!_

**Review by ashutoshgngwr — CHANGES_REQUESTED** [review]
Hi @imanudinsh 

Thank you for taking the time to translate Noice to Indonesian. I have requested a few changes. Please have a look. In addition to those changes, if you could also remove the region codes from all file paths. On Android, it will work but is not required since Indonesian is only spoken in Indonesia. In Fastlane metadata, it will not work since Google Play only recognises `id` tag. So to sum it up,

- Rename `values-in-rID` to `in`
- Rename `fastlane/metadata/android/in-ID` to `fastlane/metadata/android/id`

**Comment by imanudinsh** [comment]
Hello @ashutoshgngwr,

Thanks for reviewing my pull request, I'll fix it and come back asap

**Comment by imanudinsh** [comment]
Hi @ashutoshgngwr,

I've fixed typos, updated translations, and changed the region code on all file paths.
please let me know if it still needs improvement.

**Review by ashutoshgngwr — APPROVED** [review]
Thanks, @imanudinsh! This is great work!
